src/pages/AddPlane/AddPlane.js

```diff
@@ -0,0 +1,246 @@
+  description: Yup.string().min(2, 'Name is Too Short!').required('Description is Required'),
+  price: Yup.number().positive().integer().required('Price is required'),
+  model: Yup.string().min(2, 'model is Too Short!').required('Model is Required'),
+  year_of_manufacture: Yup.date().required('Y-O-M is required'),
+  life_span: Yup.string().required('Lifes Span is required'),
+  fees: Yup.number().positive().integer().required('Fee is required'),
+});
+
+const AddPlane = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const token = useSelector((state) => state.auth.token);
+  const planeFormData = {
+    name: '',
+    plane_type: '',
+    description: '',
+    image: '',
+    price: 0,
+    model: '',
+    year_of_manufacture: '2000-04-06',
+    life_span: '',
+    fees: 0,
+  };
+  const [planeData, setPlaneData] = useState(planeFormData);
+  const [errors, setErrors] = useState({});
+  const [imageError, setImageError] = useState('');
+
+  const handleChange = (e) => {
+    setPlaneData({ ...planeData, [e.target.name]: e.target.value });
+  };
+
+  const handelSubmit = (e) => {
+    e.preventDefault();
+    setErrors({});
+    setImageError('');
+
+    const {
+      // eslint-disable-next-line camelcase
+      name, plane_type, description, price, model, year_of_manufacture, life_span, fees,
+    } = planeData;
+
+    // To be fully implemented later
+
+    // if (isValidImageUrl(planeData.image) !== true) {
+    //   setImageError('Invalid image url');
+    //   return;
+    // }
+    CreatePlaneSchema.validate({
+      // eslint-disable-next-line camelcase
+      name, plane_type, description, price, model, year_of_manufacture, life_span, fees,
+    }, { abortEarly: false })
+      .then(() => {
+        const data = {
+          planeData,
+          token,
+        };
+
+        dispatch(createPlaneThunk(data)).then((res) => {
+          if (res.error) {
+            dispatch(setAlert(res.payload));
+          } else {
+            dispatch(setNotice('New Plane Created Successfully'));
+            navigate('/');
+          }
+        });
+      }).catch((err) => {
+        const newErrors = {};
+        err.inner.forEach((error) => {
+          newErrors[error.path] = error.message;
+        });
+        setErrors(newErrors);
+      });
+  };
+
+  return (
+    <section className="bg-gray-50 dark:bg-gray-900 ">
+      <div className="flex flex-col items-center justify-center px-6 py-8 mx-auto h-screen lg:py-0">
+        <h1 className="flex font-secondary items-center mb-6 text-2xl font-semibold text-gray-900 dark:text-white">
+          <img className="w-8 h-8 mr-2 pt-2" src="https://flowbite.s3.amazonaws.com/blocks/marketing-ui/logo.svg" alt="logo" />
+          GVASH PLANES
+        </h1>
+        <div className="w-full bg-white rounded-lg shadow dark:border md:mt-0 px-[10%] md:px-[25%] dark:bg-gray-800 dark:border-gray-700">
```

Comment:
- [ ] Kindly remove the `w-full` style

## Removing the styles in previeus comments and this one will fix your styles like the following 😉:


https://user-images.githubusercontent.com/95412943/230764686-1c26a1c8-05ec-46dd-b971-5cd35e99ecf2.mp4